packages/swap/src/event-handlers/index.ts

```diff
@@ -131,7 +131,7 @@ const handlers = [
         },
         {
           name: events[`${chain}IngressEgress`].DepositReceived,
-          handler: networkDepositReceived(chain),
+          handler: networkDepositReceived,
```

Comment:
this will add the same event handler for each chain

Reply:
because of the new args, the event handler does not need to know which chain it is

Reply:
I think we don't need the chain anymore since we are getting the `swapId` in the event data.